### PR TITLE
Feat: add configure command

### PR DESCRIPTION
**Pull request by yaronya**
### Feature Request
Add a `configure` command (similar to the `aws` cli for e.g) that gives you a prompt to modify the contents of `~/.env0/config.json`

### Solution
Use `inquerier` package to get the user's prompts and save it again to `config.json` file.

### CODE REVIEW

**Comment by yaronya** [comment]
[Add configure command](https://trello.com/c/MgNL8L4k/1683-add-configure-command)

File: node/src/config/commands.js [hunk 1]
```diff
@@ -17,6 +17,9 @@ const commands = {
     options: baseArguments,
     description: 'Cancels an deployment that is pending approval'
   },
+  configure: {
+    description: 'Configures AWS  CLI  options'
```
[the remark this answers is not in the document]
Comment (avnerenv0) [reply node/src/config/commands.js:21]: I think you mean `env0 CLI`

[the remark this answers is not in the document]
Comment (avnerenv0) [reply node/src/config/commands.js:21]: Also maybe add something about it writing a file?

[the remark this answers is not in the document]
Comment (yaronya) [reply node/src/config/commands.js]: oops, that's a shameful copy paste error 🤦 
977ea24

**Pull request merged**